keyboards/converter/adb_usb/adb_usb.h

```diff
@@ -106,4 +106,46 @@ Ported to QMK by Peter Roe <pete@13bit.me>
   { KC_NO, KC_NO, KC_NO, K7B,   KC_NO, KC_NO, KC_NO, K7F   }  \
 }
 
+/* M0118 Apple Standard Keyboard ISO
+ *                     +-------+
+ *                     | power |
+ *                     +-------+
+ * +---+---+---+---+---+---+---+---+---+---+---+---+---+-----+ +---+---+---+---+
+ * |esc| 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0 | - | = | bks | |clr| = | / | * |
+ * +---------------------------------------------------------+ +---+---+---+---+
+ * | tab | q | w | e | r | t | y | u | i | o | p | [ | ] |Ret| | 7 | 8 | 9 | + |
+ * +-----------------------------------------------------`   | +---+---+---+---+
+ * | caps | a | s | d | f | g | h | j | k | l | ; | ' | # |  | | 4 | 5 | 6 | - |
+ * +---------------------------------------------------------+ +---+---+---+---+
+ * |shif| \ | z | x | c | v | b | n | m | , | . | / |Shif| up| | 1 | 2 | 3 |   |
+ * +---------------------------------------------------------+ +-------+---|ent|
+ * |ctrl|opt  |comnd|                     |comnd |lef|rig|dwn| |   0   | . |   |
```

Comment:
Right command key is physically the same as the left one. Therefore it should be left empty here.